src/tests/java/htsjdk/samtools/CRAMEdgeCasesTest.java

```diff
@@ -1,15 +1,16 @@
-package htsjdk.samtools;
+package java.htsjdk.samtools;
```

Comment:
Package name is wrong here, which is why travis is failing -- should be just `htsjdk.samtools`